### ISSUE TITLE
Why is DS4 unstable?

### ISSUE THREAD

**Issue by saidsay-so** [CasualX/vigem-client#8]
Hi, thanks for your crate which is really helpful for my case!
I wanted to know why is the DualShock4 code considered unstable, and if there is a particular reason why the extended report code is still commented out?

**Comment by CasualX** [comment]
It's unstable because I never ended up needing a virtual dualshock4. Without practical experience I am not convinced that I've made the right API choices so I compromised by including some of the code but under a feature called unstable to indicate I might change the API (breaking) if I felt like it. That's all really.

**Comment by saidsay-so** [comment]
Do you plan to work on it sometime, or otherwise, will you accept to review a PR (#10) to add the DualShock4 support? I've started something on my side since I needed it.

**Comment by CasualX** [comment]
Yeah I don't mind reviewing your work so it can be included! Right now I'm a bit confused as you have 2 PRs, should I still look at #9 ?

**Comment by saidsay-so** [comment]
Oh yes, the other (#10) does not only add support for extended reports but also provides an API to build reports, with butttons and status handling, while #9 only adds support for extended reports. I can close #9 to focus on #10.
